src/main/java/org/folio/consortia/service/impl/SharingRoleCapabilitySetService.java

```diff
@@ -63,12 +79,12 @@ protected String getPayloadId(ObjectNode payload) {
   }
 
   @Override
-  protected String getUrl(SharingRoleCapabilitySetRequest request, HttpMethod httpMethod) {
-    String url = request.getUrl();
-    if (httpMethod.equals(HttpMethod.PUT) || httpMethod.equals(HttpMethod.DELETE)) {
-      url = url.replace("capability-sets", getConfigId(request) + "/capability-sets");
-    }
-    return url;
+  protected String getSourceValue(SourceValues sourceValue) {
+    return sourceValue.getRoleValue();
+  }
+
+  protected boolean shouldCompactRequests() {
+    return false; // payloads are different for each tenant, so requests must not be compacted
```

Comment:
can be declared in base class with false default implementation and overridden in class that needs to modify this to true, but it is as you prefer

Reply:
added some explanation in java doc about this, for me it is good to have in each tenant with some explanation